Configurations/10-main.conf

```diff
@@ -610,6 +613,7 @@ my %targets = (
         dso_scheme       => "dlfcn",
         shared_target    => "alpha-osf1-shared",
         shared_extension => ".so",
+        rpath_flag       => "-rpath ",
```

Comment:
This one should be moved to `shared-info.pl` (see @petrovr's comment)

Reply:
Well, one can actually argue both ways. Indeed, this is *compiler* flag, right? Hence it's compiler's responsibility to pass it down verbatim, or transform it to match the linker's expectations(*). In other words shared-info.pl is not necessarily right place to make this choice. Besides, since we don't know if *every* GNU ld on *every* platform supports -rpath, we formally can't add it to gnu-shared. Well, one can argue that last assertion doesn't hold water, because we effectively assume that GNU ld handles Bsymbolic and -soname=, which is effectively same "stretch" as assumption about -rpath...

(*) More prominent example is gcc's -shared option. It's same option on all platforms, but it's transformed to platform-specific options (on most platforms).